src/imitation/scripts/train_rl.py

```diff
@@ -119,22 +122,44 @@ def train_rl(
                     RuntimeWarning,
                 )
 
-        if policy_save_interval > 0:
-            save_policy_callback = serialize.SavePolicyCallback(policy_dir)
-            save_policy_callback = callbacks.EveryNTimesteps(
-                policy_save_interval,
-                save_policy_callback,
-            )
-            callback_objs.append(save_policy_callback)
-        callback = callbacks.CallbackList(callback_objs)
-
-        if agent_path is None:
-            rl_algo = rl.make_rl_algo(venv)
-        else:
-            rl_algo = rl.load_rl_algo_from_path(agent_path=agent_path, venv=venv)
-        rl_algo.set_logger(custom_logger)
-        rl_algo.learn(total_timesteps, callback=callback)
-
+        with common.make_venv(num_vec=1, log_dir=None) as eval_venv:
+            if policy_save_interval > 0:
+                save_policy_callback = serialize.SavePolicyCallback(policy_dir)
+                save_policy_callback = callbacks.EveryNTimesteps(
+                    policy_save_interval,
+                    save_policy_callback,
+                )
+                callback_objs.append(save_policy_callback)
+
+                if _config["train"]["videos"]:
```

Comment:
Here we need to init a `video_wrapper.SaveVideoCallback` instead of using `train_save_video` like others. A bit unsatisfying.

An alternative could be passing a save_video partial function into the callback.